### PR TITLE
fix(sbom): Fixes for Programming Language Vulnerabilities and SBOM Package Maintainer Details

### PR DESCRIPTION
**Pull request by santhosh1729**
1. It addresses an issue with programming language vulnerabilities by ensuring that the custom information is sent, which was previously causing a bug on the Aqua side.

2. For the SBOM, it adds the package maintainer details to the results, allowing to show this information.

### CODE REVIEW

**Comment by knqyf263** [comment]
Please mark this PR as ready for review after the tests pass.

File: rpc/common/service.proto [hunk 1]
```diff
@@ -54,6 +54,7 @@ message Package {
   string            digest      = 16;
   bool              dev         = 17;
   bool              indirect    = 18;
+  string            maintainer = 21;
```
Comment (knqyf263) [inline rpc/common/service.proto:57]: nit: Did you run `mage fmt`?

**Review by knqyf263 — COMMENTED** [review]
LGTM. Left a small comment.

File: rpc/common/service.proto (hunk 1 above)
Comment (santhosh1729) [reply rpc/common/service.proto]: No, I just did it and updated the pr, can you please check it once ? Thank you!!

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**